packages/core/src/agent/AgentConfig.ts

```diff
@@ -83,6 +83,14 @@ export class AgentConfig {
     return this.initConfig.maximumMessagePickup ?? 10
   }
 
+  public get baseMediatorReconnectionIntervalMs() {
+    return this.initConfig.baseMediatorReconnectionIntervalMs ?? 1000
```

Comment:
Okey, quick question.
Although the interval was previously set at 50, because the doubling of the interval was happening before the delay it was actually first doubling and waiting for 100ms. With my PR I have fixed that so that the baseMediator... is the actual first wait value.
Strictly speaking if we wanted the exact same behaviour it should be set at 100.
Do you want me to put it at 50 or at 100?

Reply:
I think 100 makes sense as that keeps behaviour the same